rpm/repo.yaml

```diff
@@ -1,33 +1,33 @@
 repositories:
 - arch: x86_64
-  baseurl: http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/
-  name: centos/stream8-baseos-x86_64
+  baseurl: http://mirror.stream.centos.org/9-stream/BaseOS/x86_64/os/
+  name: centos/stream9-baseos-x86_64
   gpgkey: https://www.centos.org/keys/RPM-GPG-KEY-CentOS-Official
 - arch: x86_64
-  baseurl: http://mirror.centos.org/centos/8-stream/AppStream/x86_64/os/
-  name: centos/stream8-appstream-x86_64
+  baseurl: http://mirror.stream.centos.org/9-stream/AppStream/x86_64/os/
+  name: centos/stream9-appstream-x86_64
   gpgkey: https://www.centos.org/keys/RPM-GPG-KEY-CentOS-Official
 - arch: x86_64
-  baseurl: http://mirror.centos.org/centos/8-stream/PowerTools/x86_64/os/
-  name: centos/stream8-powertools-x86_64
+  baseurl: http://mirror.stream.centos.org/9-stream/CRB/x86_64/os/
+  name: centos/stream9-crb-x86_64
   gpgkey: https://www.centos.org/keys/RPM-GPG-KEY-CentOS-Official
 - arch: x86_64
-  baseurl: https://download.copr.fedorainfracloud.org/results/sbrivio/passt/centos-stream-8-x86_64
-  name: centos/stream8-passt-x86_64
+  baseurl: https://download.copr.fedorainfracloud.org/results/sbrivio/passt/centos-stream-9-x86_64
```

Comment:
The rpms in this repo are removed once a new version is uploaded.
Please use the mirror repo  - https://github.com/kubevirt/kubevirt/pull/8403#issuecomment-1290394953

Reply:
@AlonaKaplan the RPMs from CentOS Stream itself are also going to be cycled out as new packages are introduced. That's exactly why we mirror everything. So I'm not sure we need to treat passt any different.

Moreover, I'm concerned that, if we start pointing to the passt.top mirror, @sbrivio-rh will end up having to foot a quite substantial bandwidth bill. Let's hear what he thinks about that.

Reply:
Also the COPR situation is just temporary: passt is already in Fedora, and it shouldn't be long before it is added to CentOS Stream 9 too, at which point we will be able to pick it up the same as any other package.

Reply:
> Moreover, I'm concerned that, if we start pointing to the passt.top mirror, @sbrivio-rh will end up having to foot a quite substantial bandwidth bill. Let's hear what he thinks about that.

Don't worry about that, it's not in "the cloud", I have cheap, unlimited bandwidth.

> passt is already in Fedora, and it shouldn't be long before it is added to CentOS Stream 9 too, at which point we will be able to pick it up the same as any other package.

...also true. My understanding is that we should be a few days away. The process has been going on for quite a while now.

Reply:
While we're here, note that there's a Copr issue which also affects my mirror:
* latest available build is 04992985, see e.g. https://download.copr.fedorainfracloud.org/results/sbrivio/passt/srpm-builds/04992985
* ...but that directory is unreachable from the index, see: https://download.copr.fedorainfracloud.org/results/sbrivio/passt/srpm-builds/

so my mirror can't pick it up either. That usually gets fixed in a few hours, but sometimes in about one day.

I'm trying to find a workaround. I swear I've seen an issue filed at https://pagure.io/fedora-infrastructure/issues, but I can't find it anymore. I might file a new ticket.

Reply:
Filed https://pagure.io/fedora-infrastructure/issue/10960.

Reply:
> While we're here, note that there's a Copr issue which also affects my mirror:
> 
>     * latest available build is 04992985, see e.g. https://download.copr.fedorainfracloud.org/results/sbrivio/passt/srpm-builds/04992985
> 
>     * ...but that directory is unreachable from the index, see: https://download.copr.fedorainfracloud.org/results/sbrivio/passt/srpm-builds/
> 
> 
> so my mirror can't pick it up either. That usually gets fixed in a few hours, but sometimes in about one day.

Solved now, I wasn't aware of CDN caching:
https://docs.pagure.org/copr.copr/user_documentation.html#build-succeeded-but-i-don-t-see-the-built-results

The mirror now uses non-cached sources.

Reply:
Switched to the mirror in the latest version.